.github/workflows/check-e2e-update-requirement.yaml

```
@@ -0,0 +1,150 @@
+              // skip e2e update requirement checkbox is not checked
+              // => verify e2e tests were added/updated
+              console.log('Requirement to update E2E test suite was is not set to be skipped, checking for e2e test updates in the PR...');
+              
+              const { data: files } = await github.rest.pulls.listFiles({
+                owner,
+                repo,
+                pull_number: prNumber,
+              });
+              
+              // Check if any files in tests/e2e directory were modified
+              const e2eFilesModified = files.some(file => 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)